### PR TITLE
Bump dcos-metrics

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

This brings the dcos-metrics build reference up to date with the latest state of dcos-metrics. It should help enable testing and development until the telegraf replacement PR lands. 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-39811](https://jira.mesosphere.com/browse/DCOS-39811) [Mixed Workload] Bring metrics fixes in 1.10 and 1.11 fixes to master
 - [DCOS-22194](https://jira.mesosphere.com/browse/DCOS-22194) Fix dcos-metrics pkgpanda build

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS-21663](https://jira.mesosphere.com/browse/DCOS-21663) Overhaul docs for DC/OS Metrics
 - [DCOS_OSS-2360](https://jira.mesosphere.com/browse/DCOS_OSS-2360) - Metric names can begin with numbers in dcos-metrics Prometheus endpoint
 - [INFINITY-3565](https://jira.mesosphere.com/browse/INFINITY-3565) Fix build/instructions for collector-emitter example
 - [DCOS_OSS-3304](https://jira.mesosphere.com/browse/DCOS_OSS-3304) - Metrics emitted by containers do not contain tags from labels
 - [DCOS-37452](https://jira.mesosphere.com/browse/DCOS-37452) - Increase agent collector timeout
 - [DCOS-37451](https://jira.mesosphere.com/browse/DCOS-37451) - Filter cosmos bookkeeping tags from metrics
 - [DCOS-38083](https://jira.mesosphere.com/browse/DCOS-38083) - Statsd timers are reported incorrectly by dcos-metrics
 - [DCOS-37454](https://jira.mesosphere.com/browse/DCOS-37454) - Metrics reported to prometheus inconsistent with stored metrics

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: *These changes will be entirely removed from 1.12. They are therefore not user-facing.*
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-metrics/compare/fe764a8d79fdb7a606406109f2293b86e10491cd...d5683ec1caaa5aea90b7548c09fee50bc72f160e)
  - [x] Test Results: [link to CI job test results for component](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/dcos-metrics/job/dcos-metrics-master/188/)
  - [x] Code Coverage (if available): [link to code coverage report](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/dcos-metrics/job/dcos-metrics-master/188/cobertura/)

### CODE REVIEW

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@janisz and @kensipe , mind taking another look?

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ship-it

File: packages/dcos-metrics/build [hunk 1]
```diff
@@ -35,7 +35,7 @@ mkdir -p /pkg/src/github.com/dcos
 mv /pkg/src/dcos-metrics /pkg/src/github.com/dcos/dcos-metrics
 
 pushd /pkg/src/github.com/dcos/dcos-metrics
-make build
+make rawbuild
```
Comment (janisz) [inline packages/dcos-metrics/build]: It looks odd. Usually `make build` should work for everybody

Comment (kensipe) [reply packages/dcos-metrics/build]: @philipnrmn ? ^

Comment (philipnrmn) [reply packages/dcos-metrics/build]: `make build` uses docker. Docker isn't available in pkgpanda's default build env. `rawbuild` builds without docker. This resolves [DCOS-22194](https://jira.mesosphere.com/browse/DCOS-22194), linked in the PR description.

**Comment by philipnrmn** [comment]
@mesosphere-mergebot override-status mergebot/enterprise/build-status/aggregate https://jira.mesosphere.com/browse/DCOS-37539

**Comment by philipnrmn** [comment]
@branden mind doing the override here? ^^

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status mergebot/enterprise/build-status/aggregate https://jira.mesosphere.com/browse/DCOS-37539

**Comment by jgehrcke** [comment]
@mesosphere-mergebot ship-it-label-status-check

**Pull request merged**